cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -19,7 +19,8 @@ define([
                 intro_video : null,
                 effort : null,
                 course_image_name : '',
-                course_image_asset_path : ''
+                course_image_asset_path : '',
+                pre_requisite_course : ''
```

Comment:
Add a test case below for changing this field. Note that when the Settings code was originally written, Jasmine tests were not part of the expectation. They are now, so new fields added to Settings should include unit test coverage.